packages/scss/src/components/_button.scss

```diff
@@ -398,91 +402,120 @@
 // WRAPPER
 // ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
 
+ul.button-group {
```

Comment:
I'm not sure to get why ul.button-group and button-group are splited 🤔

Reply:
I need to differentiate the legacy from the new writing with a list here to avoid the breaking change.